opm/material/fluidsystems/BlackOilFluidSystem.cpp

```diff
@@ -450,6 +455,9 @@ template <> std::vector<std::array<float, 9>> BlackOilFluidSystem<float, BlackOi
 template <> bool BlackOilFluidSystem<double, BlackOilDefaultIndexTraits>::isInitialized_ = false;
 template <> bool BlackOilFluidSystem<float, BlackOilDefaultIndexTraits>::isInitialized_ = false;
 
+template <> bool BlackOilFluidSystem<double, BlackOilDefaultIndexTraits>::useSaturatedTables_ = false;
+template <> bool BlackOilFluidSystem<float, BlackOilDefaultIndexTraits>::useSaturatedTables_ = false;
```

Comment:
you also have to add the prototype for these to shut clang up, see https://github.com/OPM/opm-common/commit/51ffb33b3876030901d156a8fc9a1bdc616bbb3e